packages/openblas/patches/0001-Add-Wno-implicit-function-declaration-flag.patch

```diff
@@ -0,0 +1,25 @@
+From 09fd1aa0aa6a98e1cebaa6e34fca1e424dab8f48 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Lo=C3=AFc=20Est=C3=A8ve?= <loic.esteve@ymail.com>
+Date: Fri, 9 Dec 2022 16:40:13 +0100
+Subject: [PATCH] Add -Wno-implicit-function-declaration flag
+
+---
+ Makefile.rule | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/Makefile.rule b/Makefile.rule
+index 5f787a9c..6890046a 100644
+--- a/Makefile.rule
++++ b/Makefile.rule
+@@ -228,7 +228,7 @@ NO_AFFINITY = 1
+ # Common Optimization Flag;
+ # The default -O2 is enough.
+ # Flags for POWER8 are defined in Makefile.power. Don't modify COMMON_OPT
+-# COMMON_OPT = -O2
++COMMON_OPT = -O2 -Wno-implicit-function-declaration -Wno-return-type
```

Comment:
Can you explain why we need this again? (Maybe in the commit message for the patch.) Is it something we can work with openblas to fix?

Reply:
I'll add the following comment in the patch:

This is needed because we are changing many signatures to return int instead of void with some regex expressions but we are not modifying the returned value which would potentially be a lot more tricky.

Reply:
What about no implicit function declaration?

Reply:
I think I removed the `-Wno-implicit-function-declaration`, maybe you are looking at an old diff?

Reply:
Ah great!